wordpress-mobile/WordPress-iOS
# P2: Add P2 branding, mark ii

Closes #15264.
PR 2 of 2 to break apart PR #15430.

This PR updates the `ReaderSiteStreamHeader` to use the P2 logo as a default site icon for P2 sites. 

### Design Review
Design review feedback incorporated from previous PR and is already approved.

### To test / Screenshots

1. Be logged in, navigate to Reader. 
2. Tap on the site heading area (the site name and site address) to be navigated to that site's list of posts.
3. Notice the default site icon changes depending on if you've navigated to a P2 site or a regular WordPress site.

| Default site icon for non-P2 sites | P2 sites light | P2 sites dark |
| --- | --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-07 at 13 34 30](https://user-images.githubusercontent.com/1062444/101401545-1d23c500-3898-11eb-9979-b418c78757b6.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-07 at 13 52 43](https://user-images.githubusercontent.com/1062444/101401553-244ad300-3898-11eb-87ae-1c9148c32ce8.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-07 at 13 52 33](https://user-images.githubusercontent.com/1062444/101401563-2745c380-3898-11eb-9cd7-cd4899ebd01a.png) |


PR submission checklist:
